Frontend/Components/Pages/DmPage.razor

```diff
@@ -1,3 +1,32 @@
 @page "/dm"
+@using Models
+@inject NavigationManager navMan
 
-<h3>this is the DM page</h3>
\ No newline at end of file
+<button class="dmChatButton" @onclick="backToMain"> Main Chat </button>
+<div class="chatMessageDiv">
```

Comment:
Could we make the area that will contain the text message a different shading?
For example discord uses a different shade for each section:
![image](https://github.com/Ldev-The-Remaster/ChattingApp/assets/60454980/717165b5-9f9e-45ce-896d-b1ec6e5c39ef)

Reply:
A different shading has been made for the message area.